xml/deploy_cephadm.xml

```diff
@@ -835,37 +839,51 @@ o- minions ................................................. [Minions: 5]
 <para>
  -->
 <screen>&prompt.cephuser;ceph orch apply mon ses-min1 ses-min2 ses-min3</screen>
+      <para>
+       and
+      </para>
+<screen>&prompt.cephuser;ceph orch apply mgr ses-min1 ses-min2 ses-min3</screen>
       <tip>
        <title>Labels for Nodes of a Specific Service</title>
        <para>
-        You can mark the nodes that will be used as MONs with the <option>mon</option> label,
-        for example:
+        You can mark the nodes that will be used as MONs or MGRs with the
+        <literal>mon</literal> or <literal>mgr</literal> label, for example:
        </para>
 <screen>
 &prompt.cephuser;ceph orch host label add ses-min1 mon
 &prompt.cephuser;ceph orch host label add ses-min2 mon
 &prompt.cephuser;ceph orch host label add ses-min3 mon
+</screen>
+
+<screen>
+&prompt.cephuser;ceph orch host label add ses-min1 mgr
+&prompt.cephuser;ceph orch host label add ses-min2 mgr
+&prompt.cephuser;ceph orch host label add ses-min3 mgr
 </screen>
        <para>
         Check the hosts and their labels:
        </para>
 <screen>
 &prompt.cephuser;ceph orch host ls
 HOST      ADDR   LABELS  STATUS
-ses-min1         mon
-ses-min2         mon
-ses-min3         mon
+ses-min1         mon,mgr
+ses-min2         mon,mgr
+ses-min3         mon,mgr
 ses-min4
 </screen>
        <para>
         Then do the actual deployment:
        </para>
+<screen>&prompt.cephuser;ceph orch apply mon label:mon</screen>
```

Comment:
I believe you wanted one to be `ceph orch apply mon` and the second to be `ceph orch apply mgr`.  In trying the command, the `label:m..` is superfluous.